tests/test_generics.py

```diff
@@ -599,3 +599,22 @@ class Model(GenericModel, Generic[AT, BT]):
         {'loc': ('a',), 'msg': 'none is not an allowed value', 'type': 'type_error.none.not_allowed'},
         {'loc': ('b',), 'msg': 'none is not an allowed value', 'type': 'type_error.none.not_allowed'},
     ]
+
+
+@skip_36
+def test_generic_subclass_of_concrete_generic():
+    T = TypeVar('T')
+    U = TypeVar('U')
+
+    class GenericBaseModel(GenericModel, Generic[T]):
+        data: T
+
+    class GenericSub(GenericBaseModel[int], Generic[U]):
```

Comment:
Not sure if this is even part of the issue or if typing even works that way but can we do stuff like:

```
class GenericSub(GenericBaseModel[U], Generic[U]):
    extra: U

GenericSub[int](data='string')  # raises validation error because data is expected to be int
```

?

Reply:
Yes, that is how I would expect it to work.